framework/src/modules/chain/init_steps/init_modules.js

```diff
@@ -0,0 +1,48 @@
+const util = require('util');
+const async = require('async');
+
+const promisifyParallel = util.promisify(async.parallel);
```

Comment:
maybe we can omit using `async` here?
I don't think it has much benefit parallelizing the initialization